baselines/diabetic_retinopathy_detection/jax_finetune_deterministic.py

```diff
@@ -167,12 +102,12 @@ def main(argv):
 
   # Get model config
   config = vit_utils.get_vit_config(
-      'deterministic', FLAGS.vit_model_size, FLAGS.pretrain_dataset)
+      'deterministic', config.vit_model_size, config.pretrain_dataset)
```

Comment:
How about removing the need for `vit_utils.get_vit_config` now that wandb can be used with the config directly?